src/backend/web/templates/suggestions/suggest_offseason_event.html

```diff
@@ -135,7 +135,7 @@ <h3>Suggest An Event</h3>
                                 <tr>
                                     <td>FIRST Sync Code</td>
                                     <td>
-                                      <input class="form-control" type="text" name="first_code" placeholder="IRI"{% if first_code %} value="{{first_code}}"{% endif %}/>
+                                      <input class="form-control" type="text" disabled="disabled" name="first_code" placeholder="Like 'IRI'.  Disabled for 2021 season."{% if first_code %} value="{{first_code}}"{% endif %}/>
```

Comment:
Could we have a Jinja if/else to disable this for 2021? We should be able to get the `current_season` in to the Jinja templates (let me know if you need help with this).

It would be great to try to improve the placeholder text, or maybe add information on the page elsewhere about what the FIRST sync code is. I'm not sure this text makes it clearer what that code is.